pulse.md

```diff
@@ -437,3 +441,255 @@ Pulse::handleExceptionsUsing(function ($e) {
+
+```js
+laravel({
+    input: [
+        'resources/css/pulse/top-sellers.css',
+        // ...
+    ],
+}),
+```
+
+You may then use the `@vite` Blade directive in your [dashboard view](#dashboard-customization), specifying the CSS entrypoint for your card:
+
+```blade
+<x-pulse>
+    @vite('resources/css/pulse/top-sellers.css')
+
+    ...
+</x-pulse>
+```
+
+<a name="custom-card-styling-css"></a>
+#### CSS Files
+
+For other use cases, including Pulse cards contained within a package, you may instruct Pulse to load additional stylesheets by defining a `css` method on your Livewire component that returns the file path to your CSS file:
+
+```php
+class TopSellers extends Card
+{
+    // ...
+
+    protected function css()
+    {
+        return __DIR__.'/../../dist/top-sellers.css';
+    }
+}
+```
+
+When this card is included on the dashboard, Pulse will automatically include the contents of this file within a `<style>` tag so it does not need to be published to the `public` directory.
+
+<a name="custom-card-styling-tailwind"></a>
+#### Tailwind CSS
+
+When using Tailwind CSS, you should create a dedicated Tailwind configuration file to avoid loading unnecessary CSS or conflicting with Pulse's Tailwind classes:
+
+```js
+export default {
+    darkMode: 'class',
+    important: '#top-sellers',
+    content: [
+        './resources/views/livewire/pulse/top-sellers.blade.php',
+    ],
+    corePlugins: {
+        preflight: false,
+    },
+};
+```
+
+You may then specify the configuration file in your CSS entrypoint:
+
+```css
+@config "../../tailwind.top-sellers.config.js";
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+You should be sure to include an `id` or `class` attribute in your cards view that matches the selector passed to Tailwind's [`important` selector strategy](https://tailwindcss.com/docs/configuration#selector-strategy):
+
+```blade
+<x-pulse::card id="top-sellers" :cols="$cols" :rows="$rows" class="$class">
+    ...
+</x-pulse::card>
+```
+
+<a name="custom-card-data"></a>
+### Data Capture & Aggregation
+
+Custom cards may fetch and display data from anywhere, however you may wish to leverage Pulse's powerful and efficient data recording and aggregation system.
+
+<a name="custom-card-data-capture"></a>
+#### Capturing Entries
+
+Pulse allows you to record "entries" which are pre-aggregated into buckets for efficient retrieval later. When using the `Pulse::record` method, you must specify a name for the type of entry you are recording and a key that determines how the aggregated data should be grouped. For most aggregation methods, you will also need to specify the value related to the individual entry. You will then need to choose one or more aggregations to perform on the entry:
```

Comment:
I found this sentence a little dificult to grok. Is Pulse doing the pre-aggregation or do I have to do it?

Reply:
Understandable! Pulse does handle it for you. I've made some tweaks - hopefully it makes a bit more sense now!